### DISCUSSION TITLE
RTX 3090 : cudaMallocAsync

### DISCUSSION THREAD

**Discussion by metamountain** [Comfy-Org/ComfyUI#2994] [Q&A]
**Cuda not accepted. Device: cuda:0**

Loading Log:

Set vram state to: HIGH_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
VAE dtype: torch.bfloat16
Using pytorch cross attention

Cuda not accepted. Device: cuda:0

How can I solve this problem ?
I guess acceleration is not working correctly?

**Comment by wangsan0315** [comment]
![222](https://github.com/comfyanonymous/ComfyUI/assets/162400393/710989dd-e455-4308-b772-09b56a983c58)
![微信截图_20240326153721](https://github.com/comfyanonymous/ComfyUI/assets/162400393/c4fc4a46-e568-4cff-bbd7-05feff24b9a1)
Same problem, can't find a solution

**Comment by kurt83340** [comment]
im not pro and maybe i will say a crap but i think everythink is ok and just
your graphic card is on the port 0 .... this number can be one if you have multiple graphic card 

so everything is ok .... according to me ... but im not programmer and all people here are smarter than me

**Comment by specblades** [comment]
Same here
